modules/pel/peltool/peltool.py

```diff
@@ -586,13 +586,25 @@ def main():
     if args.hidden:
         config.hidden = True
 
+    if args.file:
+        try:
```

Comment:
Please make it as function rather than writing a code in the main function itself.